documentation/API-authentication-token.md

```diff
@@ -1,12 +1,17 @@
 # API authentication token
 
-< to be described >
+When requesting a token on the oauth2 token resource using the client_credentials a queryparameter ‘digideliveryid’ containing the identifier for the school. A scope to access data can optionally be requested and if consent has been given, will be added as a scope in the token defined below. A tokenrequest for a token used for requesting consent for a new scope does not need to contain a requested scope.
```

Comment:
is digideliveryid not too specific? we want a schoolidentification. (digideliveryid is very dutch, and migth change, so i would prefer a more generic term.

Reply:
Please give a definition of a school before we can give the right answer.  
I think for now within the Dutch Marketed it is okay (maybe we will going to use some Dutch RIO ID someware in the time ) 

DigideliveryID is within the current chain the level where the entitlements and the learning takes place. And the place student officially lives. And there is national governance on the issuances of this ID.